frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -1,5 +1,28 @@
 import * as _ from 'lodash';
 import { Alert } from '@console/internal/components/monitoring';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+const cephStorageLabel = 'cluster.ocs.openshift.io/openshift-storage';
 
 export const filterCephAlerts = (alerts: Alert[]): Alert[] =>
   alerts.filter((alert) => _.get(alert, 'annotations.storage_type') === 'ceph');
+
+export const getFilteredPVCsData = (pvcsData: K8sResourceKind[] = []): K8sResourceKind[] =>
```

Comment:
one more thing, as these functions are generic selectors which are meant be used in other places than dashboard they should have a more descriptive names such as

```suggestion
export const getCephPVCs = (pvcs: K8sResourceKind[] = []): K8sResourceKind[] =>
```
`export const getCephPVs = (pvs: K8sResourceKind[] = []): K8sResourceKind[] =>`
`export const getCephNodes = (nodes: K8sResourceKind[] = []): K8sResourceKind[] =>`